bundle/Form/DataMapper/InformationCollectionUpdateMapper.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\DataMapper;
+
+use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
+use Netgen\Bundle\IbexaFormsBundle\Form\DataMapper;
+use Netgen\Bundle\IbexaFormsBundle\Form\DataWrapper;
+use Netgen\Bundle\IbexaFormsBundle\Form\Payload\InformationCollectionStruct;
+use RuntimeException;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\PropertyAccess\PropertyPathInterface;
+
+class InformationCollectionUpdateMapper extends DataMapper
+{
+
+    protected function mapToForm(
+        FormInterface         $form,
```

Comment:
Do not align properties like this, it's not part of our CS and it unnecessarily increases diff when something changes. Below as well.

Reply:
issue fixed in a5f8e502 commit